_store/hmr-it-android-offline-tracker.md

```diff
@@ -3,9 +3,9 @@ title: Kimai2 Android offline tracker
 name: Kimai2 Android app for offline tracking
 intro: "Kimai2 Android app for offline tracking"
 developer: hmr-it
-date: "2019-08-20 08:10:00"
+date: "2019-13-30 16:30:00"
```

Comment:
I don't think this is a valid date ^^ Can you just revert that line please?

Reply:
Oh, indeed. one month too much.

Reverted to original date